### ISSUE TITLE
ATLauncher outdated referrences

### ISSUE THREAD

**Issue by leagris** [DNSTechpack/DNS12-MC1.12.2#70]
* **Type of issue?**
  * BUG
  
* **DNS Version?**
  * 12.0.12.0
  

* **What should happen?** 
  * [ATLauncher config XML](https://download.nodecdn.net/containers/atl/packs/DNSTechpack/versions/12.0.12.0/Configs.xml)
Should not contain deprecated or invalid references to Parked squatted domain / No longer valid Discord invite
  

* **What actually happens?**
  * [ATLauncher config XML](https://download.nodecdn.net/containers/atl/packs/DNSTechpack/versions/12.0.12.0/Configs.xml)
```xml
<!--
  | ** DNS Techpack 12.x.x.x
  | ** Minecraft 1.12.2
  | ** Authors: IAmOmicron, madcock83, Zaexides
  | ** Website: http://dnstechpack.com/  ← Parked squatted domain is no longer valid
  | ** Discord: http://discord.gg/0abLGwcRy9amWzRa ← This Discord invite is no longer valid
  | -->
```
  
* **Description?** 
  * Using Item X should open Y interface but, when the item is used in the hand, the client crashes.

-----

**Comment by Madcock83** [comment]
Website links are changed. Will change discord invite when I'm back at my pc. Thank you

**Comment by leagris** [comment]
15 links persists in the ATLauncher `Config.xml`
```Bash
wget \
--quiet \
--output-document=- \
https://download.nodecdn.net/containers/atl/packs/DNSTechpack/versions/12.0.12.0/Configs.xml \
| grep \
--ignore-case \
dnstechpack.com \
| wc --lines
15
```

You can fix all referrences in the `Config.xml` with:
```Bash
wget \
--quiet \
--output-document=- \
https://download.nodecdn.net/containers/atl/packs/DNSTechpack/versions/12.0.12.0/Configs.xml \
| sed \
--regexp-extended 's/dnstechpack\.com/dnspack.weebly.com/gi;s/(discord\.gg\/)0abLGwcRy9amWzRa/\1hYxTS4g/g' \
> Config.xml
```
Then uploads the fixed `Config.xml` back to ATLauncher

**Comment by Madcock83** [comment]
As far as I know that .xml is on the ATlauncher side.  We cant change that

**Comment by robaimes** [comment]
@Madcock83 It's on things such as the title screen zip.

```xml
<mod name="DNS Title Logo" version="2.0.0" ... website="http://dnstechpack.com" hidden="yes" description="Just our logo for the title screen"/>
```

**Comment by Madcock83** [comment]
ahh.    then its not that big of a deal

**Comment by Madcock83** [comment]
Alright, should all be fixed

**Comment by leagris** [comment]
> As far as I know that .xml is on the ATlauncher side. We cant change that

https://admin.atlauncher.com/pack/DNSTechpack/versions/12.0.12.0

There you have the xml editor where you can make the needed changes or paste a replacement xml

**Comment by Madcock83** [comment]
its been done.  i didnt see anymore references to the old domain

**Comment by leagris** [comment]
```Bash
wget --quiet --output-document=- https://download.nodecdn.net/containers/atl/packs/DNSTechpack/versions/dev-1.12.2/Configs.xml | grep --ignore-case dnstechpack.com | wc --lines
4
```
→ Still 4 in your dev version of the pack

```Bash
wget --quiet --output-document=- https://download.nodecdn.net/containers/atl/packs/DNSTechpack/versions/12.0.12.0/Configs.xml | grep --ignore-case dnstechpack.com | wc --lines
15
```
→ Still 15 references in the published version 12.0.12.0

ATLauncher allow you to update the XML of published versions, so you can fix that last one in-place without publishing a new revision of your development version.
